CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Added
 
 ### Changed
+- __Action Needed__ Airflow updated to version 2.3.3; **backup flowetl_db before applying update** [#4940](https://github.com/Flowminder/FlowKit/pull/4940)
```

Comment:
I think it would be worth adding this to the docs as well - perhaps a warning [here](https://flowkit.xyz/administrator/deployment/) for anyone upgrading from version 1.16.0 or earlier.